tests/unit/fem/test_estimator.cpp

```diff
@@ -50,8 +48,302 @@ double NonsmoothSolutionZ(const mfem::Vector& x)
 {
    return std::abs(x(2)-0.5);
 }
+
+double SinXSinY(const mfem::Vector& x)
+{
+   double pi = 3.14159265358979323846;
```

Comment:
Other places in MFEM use `M_PI`, which would be a little simpler than defining `pi` as done here and below in this file.